interested-parties.md

```diff
@@ -58,6 +58,8 @@ Mambu | Adrian Mârza
 MayaData | Uma Mukkara
 MayaData | Karthik S
 Microsoft | Chris Sanders
+Morgan Stanley | Erick Bourgeois
+Morgan Stanley | Graeme Hay
```

Comment:
Thanks @ebourgeois and @grmhay — A tip similar to https://github.com/gitops-working-group/gitops-working-group/pull/170#pullrequestreview-780057521
> Hi! Welcome! Could you add your GitHub username next to your name - we had not done this with previous interested parties and then realized we really had no efficient way to be able to reach out to all of you so I want to make sure we start fixing that ;)